src/modules/services/services.conf

```diff
@@ -1,20 +1,24 @@
 ---
-#systemd services and targets are enabled in this precise order
+# systemd services and targets example:
 
-services:
-  - name: "NetworkManager"  #name of the service file
-    mandatory: false        #true=> if enabling fails the installer errors out and quits
-                            #false=>if enabling fails print warning to console and continue
-  - name: "cups"
-    mandatory: false
-
-targets:
-  - name: "graphical"
-    mandatory: true
-
-disable:
-  - name: "pacman-init"
-    mandatory: false
+# services:
+#   - name: "NetworkManager"  # name of the service file
+#     mandatory: false        # true=> if enabling fails the installer errors out and quits
+#                             # false=>if enabling fails print warning to console and continue
```

Comment:
Those three end-of-line comments should be above, describing what entries / keys are needed in each of the lists.